website/pages/en/developing/developer-faqs.mdx

```diff
@@ -55,8 +55,9 @@ Yes. On `graph init` command itself you can add multiple datasources by entering
 ## 11. I want to contribute or add a GitHub issue. Where can I find the open source repositories?
 
 - [graph-node](https://github.com/graphprotocol/graph-node)
-- [graph-cli](https://github.com/graphprotocol/graph-cli)
-- [graph-ts](https://github.com/graphprotocol/graph-ts)
+- [graph-tooling](https://github.com/graphprotocol/graph-tooling)
+- [graph-docs](https://github.com/graphprotocol/docs)
```

Comment:
Technically the name of the repo is `docs`, not `graph-docs`, but the list looks nicer this way so... 👍